pkg/test/resourcefixture/testdata/basic/spanner/v1beta1/spannerdatabase/_http.log

```diff
@@ -0,0 +1,378 @@
+User-Agent: Terraform/ (+https://www.terraform.io) Terraform-Plugin-SDK/2.10.1 terraform-provider-google-beta/kcc/controller-manager
+
+404 Not Found
+Cache-Control: private
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "error": {
+    "code": 404,
+    "message": "database \"projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test\" not found",
+    "status": "NOT_FOUND"
+  }
+}
+
+---
+
+POST https://spanner.googleapis.com/v1/projects/${projectId}/instances/spannerinstance-${uniqueId}/databases?alt=json
+Content-Type: application/json
+User-Agent: Terraform/ (+https://www.terraform.io) Terraform-Plugin-SDK/2.10.1 terraform-provider-google-beta/kcc/controller-manager
+
+{
+  "createStatement": "CREATE DATABASE `spannerdatabase-test`"
+}
+
+200 OK
+Cache-Control: private
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "metadata": {
+    "@type": "type.googleapis.com/google.spanner.admin.database.v1.CreateDatabaseMetadata",
+    "database": "projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test"
+  },
+  "name": "projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test/operations/${operationID}"
+}
+
+---
+
+GET https://spanner.googleapis.com/v1/projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test/operations/${operationID}?alt=json
+Content-Type: application/json
+User-Agent: Terraform/ (+https://www.terraform.io) Terraform-Plugin-SDK/2.10.1 terraform-provider-google-beta/kcc/controller-manager
+
+200 OK
+Cache-Control: private
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "done": true,
+  "metadata": {
+    "@type": "type.googleapis.com/google.spanner.admin.database.v1.CreateDatabaseMetadata",
+    "database": "projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test"
+  },
+  "name": "projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test/operations/${operationID}",
+  "response": {
+    "@type": "type.googleapis.com/google.spanner.admin.database.v1.Database",
+    "createTime": "2024-04-01T12:34:56.123456Z",
+    "databaseDialect": "GOOGLE_STANDARD_SQL",
+    "name": "projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test",
+    "state": "READY",
+    "versionRetentionPeriod": "1h"
+  }
+}
+
+---
+
+PATCH https://spanner.googleapis.com/v1/projects/${projectId}/instances/spannerinstance-${uniqueId}/databases/spannerdatabase-test/ddl?alt=json
```

Comment:
Maybe I miss something. 
I don't see a update.yaml file under https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/test/resourcefixture/testdata/basic/spanner/v1beta1/spannerdatabase/
How this patch is triggered?

Reply:
This is an implementation detail of the terraform controller. I think it works around the issue that it's impossible to specify extra DDL statements for Postgres-flavor databases in the create call by just always taking two steps (even if it's a GOOGLESQL-flavor database).